tests/e2e/specs/miscellaneous/VsixInstallationDisableTest.spec.ts

```diff
@@ -0,0 +1,283 @@
+	}
+
+	// ========== Test Execution ==========
+
+	suiteSetup('Login to cluster and setup initial state', function (): void {
+		kubernetesCommandLineToolsExecutor.loginToOcp();
+		cleanupConfigMaps();
+	});
+
+	suiteSetup('Login to application', async function (): Promise<void> {
+		await loginTests.loginIntoChe();
+	});
+
+	test('Apply ConfigMaps that disable VSIX and set default extensions', function (): void {
+		applyConfigMap(CONFIG_FILES.DISABLE_VSIX);
+		applyConfigMap(CONFIG_FILES.DEFAULT_EXTENSIONS);
+	});
+
+	test('Create and open workspace from Git repository', async function (): Promise<void> {
+		await workspaceHandlingTests.createAndOpenWorkspaceFromGitRepository(TEST_REPO_URL);
+		await workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
+		workspaceName = WorkspaceHandlingTests.getWorkspaceName();
+		registerRunningWorkspace(workspaceName);
+	});
+
+	test('Wait workspace readiness', async function (): Promise<void> {
+		await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+	});
+
+	test('Verify VSIX disabled notifications', async function (): Promise<void> {
+		const hasDisableNotification: boolean = await notificationHandler.checkForNotification('install from vsix command is disabled');
+		expect(hasDisableNotification).to.be.true;
+	});
+
+	test('Perform trust author dialog', async function (): Promise<void> {
+		await projectAndFileTests.performTrustAuthorDialog();
+	});
+
+	test('Verify VSIX installation is disabled in UI', async function (): Promise<void> {
+		await checkVsixInstallationInUI(false);
+	});
+
+	test('Verify default extensions are not auto-installed when VSIX disabled', async function (): Promise<void> {
+		await checkDefaultExtensionsInstallation(false);
+	});
+
+	test('Enable VSIX installation and create new workspace', async function (): Promise<void> {
+		// clean up current workspace
+		await dashboard.openDashboard();
+		await testWorkspaceUtil.stopAndDeleteWorkspaceByName(workspaceName);
+		registerRunningWorkspace('');
+
+		// apply new ConfigMaps
+		applyConfigMap(CONFIG_FILES.ENABLE_VSIX);
+		applyConfigMap(CONFIG_FILES.DEFAULT_EXTENSIONS);
+
+		Logger.info('Waiting for new ConfigMap settings to take effect...');
+		await driverHelper.wait(30000);
+
+		// create new workspace
+		await workspaceHandlingTests.createAndOpenWorkspaceFromGitRepository(TEST_REPO_URL);
+		await workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
+		workspaceName = WorkspaceHandlingTests.getWorkspaceName();
+		registerRunningWorkspace(workspaceName);
+	});
+
+	test('Wait workspace readiness', async function (): Promise<void> {
+		await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+	});
+
+	test('Verify default extension installation success notifications', async function (): Promise<void> {
+		const successTexts: string[] = ['Completed installing extension', 'installed', 'extension'];
```

Comment:
It seems to be overly complicated, IMHO.
Shouldn't the "Completed installing extension" notification be enough to check for?